client/auth/action-creators.ts

```diff
@@ -132,10 +133,27 @@ export function verifyEmail(token: string) {
   return idRequest('@auth/verifyEmail', () => fetch<void>(url, { method: 'post' }))
 }
 
-export function sendVerificationEmail() {
-  const url = '/api/1/users/sendVerification'
-
-  return idRequest('@auth/sendVerificationEmail', () => fetch<void>(url, { method: 'post' }))
+export function sendVerificationEmail(): ThunkAction {
+  return dispatch =>
+    fetch<void>('/api/1/users/sendVerification', { method: 'post' }).then(
+      () =>
+        dispatch(
+          openSnackbar({
+            message: 'Verification email has been sent successfully.',
+            time: TIMING_LONG,
+          }),
+        ),
+      err => {
+        dispatch(
+          openSnackbar({
+            message:
+              'Something went wrong while sending a verification email, please try again later.',
+            time: TIMING_LONG,
+          }),
+        )
+        throw err
```

Comment:
Probably don't wanna throw this, it'll count as an uncaught error (and you *did* handle it here)